_docs/pages/supported-tracers.md

```diff
@@ -6,30 +6,12 @@
 
 ## Jaeger
 
-[Jaeger \ˈyā-gər\](http://uber.github.io/jaeger) is Uber's distributed tracing system, built with OpenTracing support from inception. Jaeger includes OpenTracing client libraries in several languages: [Java](https://github.com/uber/jaeger-client-java), [Go](https://github.com/uber/jaeger-client-go), [Python](https://github.com/uber/jaeger-client-python), and [Node.js](https://github.com/uber/jaeger-client-node).
+[Jaeger \ˈyā-gər](http://uber.github.io/jaeger) is Uber's distributed tracing system, built with OpenTracing support from inception. Jaeger includes OpenTracing client libraries in several languages: [Java](https://github.com/uber/jaeger-client-java), [Go](https://github.com/uber/jaeger-client-go), [Python](https://github.com/uber/jaeger-client-python), and [Node.js](https://github.com/uber/jaeger-client-node).
```

Comment:
@yurishkuro ah :)

Ok, look again. My local `.md` syntax highlighter is more pedantic than jekyll about the escaping, it seems. What I have in the latest revision pleases both the highlighter and jekyll.